docs/classes/asyncapidiff.AsyncAPIDiff.md

```diff
@@ -0,0 +1,116 @@
+[@asyncapi/diff](../README.md) / [Modules](../modules.md) / [asyncapidiff](../modules/asyncapidiff.md) / AsyncAPIDiff
+
+# Class: AsyncAPIDiff
+
+[asyncapidiff](../modules/asyncapidiff.md).AsyncAPIDiff
+
+Implements methods to deal with diff output.
+
+**`returns`**
```

Comment:
Not sure what this `return` signifies here? Maybe it's missing the return type?

Reply:
It returns AsynAPIDiff itself. Fixed it !